x/ibc/core/02-client/handler_test.go

```diff
@@ -204,9 +205,7 @@ func (suite *ClientTestSuite) TestUpgradeClient() {
 
 		tc.setup()
 
-		_, err := client.HandleMsgUpgradeClient(
-			suite.chainA.GetContext(), suite.chainA.App.IBCKeeper.ClientKeeper, msg,
-		)
+		_, err := keeper.Keeper.UpgradeClient(*suite.chainA.App.IBCKeeper, sdk.WrapSDKContext(suite.chainA.GetContext()), msg)
```

Comment:
`TestUpgradeClient` in `02-client/handler_test.go` should be moved to `x/ibc/core/keeper/msg_server_test.go`